public/app/routes/routes.ts

```diff
@@ -544,6 +544,16 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       controllerAs: 'ctrl',
       reloadOnSearch: false,
     })
+    .when('/goto/:shortLinkUid', {
```

Comment:
Also considered linking to /api/goto/:uid and having that endpoint return a 302, but it wasn't clear how to provide decent error handling in that scenario